### DISCUSSION TITLE
How can I put pictures side by side

### DISCUSSION THREAD

**Discussion by guangkunxiong** [toeverything/AFFiNE#4851] [Q&A]
I wish I could arrange the pictures horizontally。Thanks

**Comment by lawvs** [comment]
In the new version(after `0.10.0-canary.10`), you can insert edgeless pictures, and add them as a ref frame.


https://github.com/toeverything/AFFiNE/assets/18554747/b1448fa5-4338-4c73-b00c-c7b0e70a85e2
